docs/UserGuide.md

```diff
@@ -96,14 +96,37 @@ Following the usage example above we would see the following message:
 Note: If any of the non-optional parameters `Name`, `Current price` and `Sentiment` are provided with invalid
 inputs, you would be prompted to give a valid input.
 
+## Editing an instrument: `edit`
+Edit an existing instrument.
 
+Format: `edit`
 
+Upon entering the command, a sequence of prompts would be displayed.
+1. Enter the instrument index.
+2. Choose the parameter to edit.
+
+Example usage
+```
+mTracker$> edit
+	Please key in the index of instrument: 
+mTracker$> 1
+	Please key in index of the parameter to edit:
+		1) Name
+		2) Current Price
+		3) Sentiments
+		4) Remarks
```

Comment:
Instructions here should be "Please enter one or more " + instrument.getType() + " parameters to edit." + instrument.getListOfParameters());

example: "Please enter one or more stock parameters to edit. (Accepted parameters: name, currentprice, sentiment and remarks)

Then the user can give a string like this: "name currentprice" for eg, not the index